src/applications/caregivers/definitions/sharedSchema.js

```diff
@@ -0,0 +1,32 @@
+import { merge, omit } from 'lodash';
+import {
+  yesNoSchema,
+  addressNoMilitarySchema,
+  fullNameSchema as platformFullNameSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+// declare schema for use with non-autofill address definitions
+export const addressSchema = definition =>
```

Comment:
We need to override the platform web component pattern schema with our own schema definition. The override takes care of things like setting min and max character counts and updating regex patterns.